### PR TITLE
Improve mdadm and mdadm.conf manuals

### PR DESCRIPTION
**Pull request by Paiusco**
The `--udev-rules` flag was barely mentioned under manuals, so I added a bit into both.

### CODE REVIEW

File: mdadm.8.in [hunk 1]
```diff
@@ -1830,6 +1830,17 @@ can be found it
...
 
+.TP
+.B \-\-udev\-rules=
+Given the policies defined under
+.IR mdadm.conf
+it generates the udev rules to handle hot-plug bare devices
+
+See
+.BR mdadm.conf (5)
+for more details about policies.
```
Comment (mtkaczyk) [inline mdadm.8.in:1841]: Please use keyword "POLICY" where possible. For less experienced reader it is easier to catch up if you are referring to the same context (and simple search by POLICY will work in mdadm.conf), for example:

```
Given the POLICY line/lines are defined under
.IR {CONFFILE}
```
{CONFFILE} can be used to set primary conf file location instead, if you want to. Not an issue for sure.

so far I remember `.IR` does not add something special but I could be wrong here. I'm not really oriented in man syntax.

Comment (mtkaczyk) [reply mdadm.8.in:1841]: Please also mention that the "usage example" can be found in man mdadm.conf.

File: mdadm.8.in [hunk 2]
```diff
@@ -1830,6 +1830,17 @@ can be found it
...
+
+
```
Comment (mtkaczyk) [inline mdadm.8.in:1843]: Do we need double space here?

**Review by mtkaczyk — CHANGES_REQUESTED** [review]
Thanks for patches. The changes looks good.
Please review submitting guide in Readme.
You have to add description and sign-off your commits.

In one commit message, please refer specify to the option you described `Explain --udev-rules option`


```
[1/2] Check commit - b79b4146be1c81ad3cc5ac8af2d09239f606eecd
-------------------------------------------------------------
WARNING:COMMIT_MESSAGE: Missing commit description - Add an appropriate one
Error: WARNING:COMMIT_MESSAGE: Missing commit description - Add an appropriate one


ERROR:MISSING_SIGN_OFF: Missing Signed-off-by: line(s)
Error: ERROR:MISSING_SIGN_OFF: Missing Signed-off-by: line(s)


total: 1 errors, 1 warnings, 42 lines checked
>> Failure

-------------------------------------------------------------
[2/2] Check commit - 50cba0b7ac4f1e[138](https://github.com/md-raid-utilities/mdadm/actions/runs/11202483091/job/31155296752?pr=105#step:5:139)e2f1037cf364addfb85bf8a
-------------------------------------------------------------
WARNING:COMMIT_MESSAGE: Missing commit description - Add an appropriate one
Error: WARNING:COMMIT_MESSAGE: Missing commit description - Add an appropriate one


ERROR:MISSING_SIGN_OFF: Missing Signed-off-by: line(s)
Error: ERROR:MISSING_SIGN_OFF: Missing Signed-off-by: line(s)

```

File: mdadm.8.in (hunk 2 above)
Comment (Paiusco) [reply mdadm.8.in:1842]: deleted

File: mdadm.8.in (hunk 1 above)
Comment (Paiusco) [reply mdadm.8.in]: added the {CONFFILE} and moved from `policies` to `POLICY`

**Review by mtkaczyk — APPROVED** [review]

**Review by ktanska — APPROVED** [review]

**Pull request merged**